dist/standalone-galleon-pack/src/main/resources/modules/system/layers/base/org/jboss/prospero-dep/main/module.xml

```diff
@@ -26,7 +26,6 @@
         <artifact name="${commons-io:commons-io}"/>
         <artifact name="${commons-codec:commons-codec}"/>
         <artifact name="${org.apache.commons:commons-lang3}"/>
-        <artifact name="${org.wildfly.installation-manager:installation-manager-api}"/>
```

Comment:
This artifact should not be necessary for Prospero runtime work, it's only necessary when WildFly requires Prospero via the SPI.

If we do not follow this approach (removed in `org.jboss.prospero-dep` but added in `org.jboss.prospero`), we will get two artifacts under the Jboss Modules classpath, the one incorporated by Prospero Galleon Pack, and the one incorporated by WildFly, is in turn, necessary to allow WildFly to load the SPI.

Reply:
@spyrkob as we have discussed, I've removed the module.xml changes and now we will use the `org.wildfly.prospero:prospero-wildfly-galleon-pack` instead of `org.wildfly.prospero:prospero-standalone-galleon-pack` for the WildFly integration.

From now we assume `org.wildfly.prospero:prospero-standalone-galleon-pack` will never work properly with the WildFly Integration